x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/function/scalar/string/StartsWithFunctionProcessor.java

```diff
@@ -20,43 +20,51 @@
 
     private final Processor source;
     private final Processor pattern;
+    private final boolean isCaseSensitive;
 
-    public StartsWithFunctionProcessor(Processor source, Processor pattern) {
+    public StartsWithFunctionProcessor(Processor source, Processor pattern, boolean isCaseSensitive) {
```

Comment:
Maybe remove the `Function` as we don't normally use it in processor names.

Reply:
There are functions where this pattern is used - String functions.